satpy/modifiers/atmosphere.py

```diff
@@ -77,6 +115,13 @@ def __call__(self, projectables, optional_datasets=None, **info):
             refl_cor_band = corrector.get_reflectance(sunz, satz, ssadiff,
                                                       vis.attrs['wavelength'][1],
                                                       red.data)
+
+        if lim_low > lim_high:
+            lim_low = lim_high
+        logger.info("Reducing Rayleigh correction at high zenith angles by strength %s.", strength)
```

Comment:
I would probably remove this, since a similar logging message will come form pyspectral: https://github.com/pytroll/pyspectral/blob/main/pyspectral/rayleigh.py#L294.